src/theme/ItaliaTheme/Subsites/ItaliaTheme/Blocks/_form.scss

```diff
@@ -0,0 +1,50 @@
+@use '../../all_variables' as *;
+
+@mixin block(
+  $subsite-primary,
+  $subsite-primary-text,
+  $subsite-secondary,
+  $subsite-light-theme
+) {
+  .form,
+  & {
+    .form-check {
+      [type='checkbox'],
+      [type='radio'] {
+        &:checked + label {
+          &::after {
+            @if $subsite-light-theme {
+              border-color: $subsite-secondary;
+              background-color: $subsite-secondary;
+            } @else {
+              border-color: $subsite-primary;
+              background-color: $subsite-primary;
+            }
+            z-index: 0;
```

Comment:
Perché questo serve solo per i sottositi? È un fix che potrebbe servire per tutto?

Reply:
In realtà quello stile c'è già, dev'essere stata una svista mentre copiavo gli stili.
Correggo.